blocks/init/src/blocks/custom/columns/components/columns-responsive-tab-content.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { SelectControl } from '@wordpress/components';
+
+export const columnsGutterVertical = [
+  { label: __('Default (25px)', 'eightshift-boilerplate'), value: 'default' },
+  { label: __('Big (50px)', 'eightshift-boilerplate'), value: 'big' },
+  { label: __('None (0px)', 'eightshift-boilerplate'), value: 'none' },
```

Comment:
The ordering is ticking my ocd, could they go in ascending fashion 0, 25, 50? (even tho the default one would be in the middle)